src/config/transform.rs

```diff
@@ -193,6 +193,7 @@ pub trait TransformConfig: DynClone + NamedComponent + core::fmt::Debug + Send +
     /// of events flowing through the transform.
     fn outputs(
         &self,
+        enrichment_tables: enrichment::TableRegistry,
```

Comment:
It seems odd that every component is not getting some enrichment table, what's the reasoning behind that?

Reply:
Ah, got it - I remembered you mentioning the overloading, but wasn't sure exactly how that was happening. Thanks!